packages/@aws-cdk/aws-cloudwatch/README.md

```diff
@@ -24,6 +26,49 @@ represents the amount of errors reported by that Lambda function:
 const errors = fn.metricErrors();
 ```
 
+### Instantiating a new Metric object
+
+If you want to reference a metric that is not yet exposed by an existing construct,
+you can instantiate a `Metric` object to represent it. For example:
+
+```ts
+const metric = new Metric({
+    namespace: 'MyNamespace',
+    metricName: 'MyMetric',
+    dimensions: {
+        ProcessingStep: 'Download'
+    }
+});
+```
+
+### Metric Math
+
+Math expressions are supported by instantiating the `MathExpression` class.
+For example, a math expression that sums two other metrics looks like this:
+
+```ts
+const allProblems = new MathExpression({
+    expression: "errors + faults",
+    usingMetrics: {
+        errors: myConstruct.metricErrors(),
+        faults: myConstruct.metricFaults(),
+    }
+})
```

Comment:
Do you think it might be possible to also provide the option to simply embed metric objects (via tokens) into the expression like so:

```ts
expression: `${myConstruct.metricErrors()} + ${myConstruct.metricFaults()}`
```

I feel this might be a much more intuitive API for most users... And less error prone.